src/index.js

```diff
@@ -0,0 +1,76 @@
+import './assets/style.scss';
+import './assets/bg-img.jpg';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+import dom from './dom';
+import api from './weatherAPI';
+import tempConverter from './tempConverter';
+
+document.body.appendChild(dom.createNav());
+document.body.appendChild(dom.createMainContentContainer());
+document.body.appendChild(dom.createFooter());
+
+const mainContentDiv = document.querySelector('.main-content');
+mainContentDiv.appendChild(dom.displayInstructionText());
+
+function clearMainContentDiv() {
+  while (mainContentDiv.lastChild) {
+    mainContentDiv.lastChild.remove();
+  }
+}
+
+function convertTemp(element) {
+  const temp = document.getElementById('temp');
+
+  if (temp.getAttribute('data-temp') === 'kelvin') {
+    temp.setAttribute('data-temp', 'celcius');
+    element.textContent = 'Convert to K';
+
+    const kelvinTemp = api.dataObj.temp;
+    const kelvinMinTemp = api.dataObj.minTemp;
+    const kelvinMaxTemp = api.dataObj.maxTemp;
+
+    const celciusTemp = tempConverter(kelvinTemp);
+    const celciusMinTemp = tempConverter(kelvinMinTemp);
+    const celciusMaxTemp = tempConverter(kelvinMaxTemp);
+
+    temp.textContent = celciusTemp;
+    document.getElementById('minTemp').textContent = celciusMinTemp;
+    document.getElementById('maxTemp').textContent = celciusMaxTemp;
+  } else {
+    temp.setAttribute('data-temp', 'kelvin');
+    element.textContent = 'Convert to °C';
+
+    const kelvinTemp = `${api.dataObj.temp}K`;
```

Comment:
- [**optional**] You should consider removing this method and adding it to a utils or helper file, then export and import it in the index.js file and use where necessary. It is ideal that your index.js should only be responsible for piecing together the logic or features from other imported files to make up a working app.